01-intro-to-java/lab/03-forest-terrain/SampleForestTrackerTestSuite.java

```diff
@@ -0,0 +1,33 @@
+import java.util.Arrays;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class SampleForestTrackerTestSuite {
```

Comment:
I guess it is only a test case (SampleForestTrackerTest), not a suite that includes many test classes.
https://github.com/junit-team/junit4/wiki/categories